<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exit_window</class>
 <widget class="QWidget" name="exit_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="central_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="backgroundcolor_frame">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(29, 53, 87);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="content_frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <widget class="QComboBox" name="productos_exit">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>60</y>
              <width>151</width>
              <height>21</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox {
        background-color: #F1FAEE;
        color: #1D3557;
        border: 2px solid #457B9D;
        border-radius: 5px;
        padding: 5px;
        font-size: 14px;
    }
    QComboBox::drop-down {
        border: none;
    }
    QComboBox::down-arrow {
        image: url(C:/Users/velar/OneDrive/Escritorio/Planta2/assets/icons8-expand-arrow-24.png);
    }
    QComboBox QAbstractItemView {
        border: 2px solid #457B9D;
        selection-background-color: #457B9D;
    }</string>
            </property>
           </widget>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>20</y>
              <width>91</width>
              <height>31</height>
             </rect>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
        color: #ffffff;            /* Color del texto */
        
        border: 2px solid #457B9D; /* Color del borde */
        border-radius: 5px;        /* Radio del borde (para bordes redondeados) */
        padding: 5px;              /* Espacio interior alrededor del texto */
        font-size: 14px;           /* Tamaño del texto */
    }</string>
            </property>
            <property name="text">
             <string>PRODUCTO</string>
            </property>
           </widget>
           <widget class="QDoubleSpinBox" name="cantidad_exit_spin">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>140</y>
              <width>81</width>
              <height>41</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true"> QDoubleSpinBox {
        background-color: #F1FAEE;
        color: #1D3557;
        border: 2px solid #457B9D;
        border-radius: 5px;
        padding: 5px;
        font-size: 14px;
    }
    QDoubleSpinBox::up-button {
        width: 20px;
        border: none;
        background: #457B9D url(C:/Users/velar/OneDrive/Escritorio/Planta2/assets/icons8-collapse-arrow-24.png) no-repeat center;
    }
    QDoubleSpinBox::down-button {
        width: 20px;
        border: none;
        background: #457B9D url(C:/Users/velar/OneDrive/Escritorio/Planta2/assets/icons8-expand-arrow-24.png) no-repeat center;
    }
    QDoubleSpinBox::up-button:pressed {
        background: #1D3557 url(path_to_up_arrow) no-repeat center;
    }
    QDoubleSpinBox::down-button:pressed {
        background: #1D3557 url(path_to_down_arrow) no-repeat center;
    }</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>70</x>
              <y>100</y>
              <width>91</width>
              <height>31</height>
             </rect>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
        color: #ffffff;            /* Color del texto */
        
        border: 2px solid #457B9D; /* Color del borde */
        border-radius: 5px;        /* Radio del borde (para bordes redondeados) */
        padding: 5px;              /* Espacio interior alrededor del texto */
        font-size: 14px;           /* Tamaño del texto */
    }</string>
            </property>
            <property name="text">
             <string>CANTIDAD</string>
            </property>
           </widget>
           <widget class="QPushButton" name="submit_exit_button">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>200</y>
              <width>101</width>
              <height>31</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
        background-color: #457B9D;
        color: #F1FAEE;
        border: 2px solid #1D3557;
        border-radius: 5px;
        padding: 5px;
        font-size: 14px;
    }
    QPushButton:hover {
        background-color: #1D3557;
        border: 2px solid #457B9D;
    }
    QPushButton:pressed {
        background-color: #1D3557;
        border: 2px solid #457B9D;
        padding-left: 7px;  /* Para dar un efecto de &quot;presión&quot; */
    }</string>
            </property>
            <property name="text">
             <string>Aplicar</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
